rs/sns/init/src/lib.rs

```diff
@@ -3432,36 +3401,40 @@ mod test {
     }
 
     #[test]
-    fn test_number_of_sns_neuorns_for_direct_participants_cannot_exceed_limit() {
+    fn test_validate_participation_constraints() {
+        // Common part for the happy and failing scenarios.
+        let fdvp = FractionalDVP {
+            swap_distribution: Some(SwapDistribution {
+                initial_swap_amount_e8s: MAX_DIRECT_ICP_CONTRIBUTION_TO_SWAP,
+                // Not used in this test.
+                total_e8s: 0,
+            }),
+            // Not used in this test.
+            developer_distribution: None,
+            treasury_distribution: None,
+            airdrop_distribution: None,
+        };
+        let sns_init_payload = SnsInitPayload {
+            max_direct_participation_icp_e8s: Some(MAX_DIRECT_ICP_CONTRIBUTION_TO_SWAP),
+            min_direct_participation_icp_e8s: Some(1),
+            max_participant_icp_e8s: Some(MAX_DIRECT_ICP_CONTRIBUTION_TO_SWAP),
+            min_participants: Some(40_000),
+            initial_token_distribution: Some(FractionalDeveloperVotingPower(fdvp)),
+            neuron_basket_construction_parameters: Some(NeuronBasketConstructionParameters {
+                count: 2,
+                // Not used in this test.
+                dissolve_delay_interval_seconds: 0,
+            }),
+            neuron_minimum_stake_e8s: Some(1),
+            transaction_fee_e8s: Some(0),
+            ..SnsInitPayload::with_valid_values_for_testing_pre_execution()
+        };
         // Happy scenario: Test that if the constraints are satisfied, the validation passes.
         {
-            let fdvp = FractionalDVP {
-                swap_distribution: Some(SwapDistribution {
-                    initial_swap_amount_e8s: MAX_DIRECT_ICP_CONTRIBUTION_TO_SWAP,
-                    // Not used in this test.
-                    total_e8s: 0,
-                }),
-                // Not used in this test.
-                developer_distribution: None,
-                treasury_distribution: None,
-                airdrop_distribution: None,
-            };
             let sns_init_payload = SnsInitPayload {
-                max_direct_participation_icp_e8s: Some(MAX_DIRECT_ICP_CONTRIBUTION_TO_SWAP),
-                min_direct_participation_icp_e8s: Some(1),
                 // This value being high enough ensures there wouldn't be too many SNS neurons.
```

Comment:
We no longer use the `min_participant_icp_e8s` to limit the number of participants, right?

Reply:
Correct, fixed it